### ISSUE TITLE
All external test builds failed with 'No space left on device' on test-scaleway-ubuntu1604-x64-1

### ISSUE THREAD

**Issue by sophia-guo** [AdoptOpenJDK/openjdk-tests#1103]
All container test builds failed on test-scaleway-ubuntu1604-x64-1 with message 

```
20:27:05  ERROR: [ssh-agent] Could not register BouncyCastle on the remote agent. 20:27:05  java.nio.file.FileSystemException: /tmp/resource-1012398343128323010: No space left on device 20:27:05  	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:91)  
```
https://ci.adoptopenjdk.net/view/Test_external/
https://ci.adoptopenjdk.net/view/Test_external/ https://ci.adoptopenjdk.net/view/Test_external/job/openjdk11_hs_external_extended_elasticsearch_x86-64_linux/28/console

 We had exactly same issue before on different machine  test-softlayer-ubuntu1604-x64-1 https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/761

If dangling docker volumes are the real reason of this issue we may need to consider the story of docker volume clean up.

**Comment by sophia-guo** [comment]
'If your container generates non-persistent state data, consider using a tmpfs mount to avoid storing the data anywhere permanently, and to increase the container’s performance by avoiding writing into the container’s writable layer.'  https://docs.docker.com/storage/volumes/

**Comment by sophia-guo** [comment]
I will log in the docker machines and do a cleanup manually cleanup and then the issue will be closed by #1104 , which will avoid to create any new dangling docker volumes from test builds.

As mentioned in https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/777#issuecomment-487678617 those dangling docker volumes should not be the reason of 'no space left'. The further investigation will be continued in https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/761
